install/installer/pkg/common/render.go

```diff
@@ -123,10 +124,11 @@ func NewRenderContext(cfg config.Config, versionManifest versions.Manifest, name
 	cfg.Experimental = nil
 
 	ctx := &RenderContext{
-		Config:             cfg,
-		VersionManifest:    versionManifest,
-		Namespace:          namespace,
-		experimentalConfig: us,
+		Config:                cfg,
+		VersionManifest:       versionManifest,
+		Namespace:             namespace,
+		InstallationShortname: cfg.Metadata.InstallationShortname,
```

Comment:
Is there a reason why you put this at the root level rather than getting it from the `Config` object?

Reply:
Ah, no, just overlooked that. Will change.

Reply:
Don't worry - I did the same as well